test/stabilizer_gates.jl

```diff
@@ -48,6 +48,7 @@ const one_qubit_test_cases =
      ('-', Y, [1 0 0]),
      ('-', Z, [1 0 0]),
      ('-', H, [0 1 2]))
+     # Should really add all the new one-qubit gates here
```

Comment:
Why not? Is it very difficult to do so?

Reply:
I will remove the comment - it's much easier to do it the way I ended up doing it, by checking that the results are the same as the sequence of decomposed operations.